src/SFA.DAS.Campaign/SFA.DAS.Campaign.csproj

```diff
@@ -5,16 +5,18 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <ProjectReference Include="..\SFA.DAS.SharedOuterApi\SFA.DAS.SharedOuterApi.csproj" />
+      <Compile Remove="Application\Services\**" />
+      <EmbeddedResource Remove="Application\Services\**" />
+      <None Remove="Application\Services\**" />
     </ItemGroup>
 
     <ItemGroup>
-      <PackageReference Include="contentful.aspnetcore" Version="6.0.7" />
-      <PackageReference Include="MediatR" Version="6.0.0" />
+      <ProjectReference Include="..\SFA.DAS.SharedOuterApi\SFA.DAS.SharedOuterApi.csproj" />
     </ItemGroup>
 
     <ItemGroup>
-      <Folder Include="Application\Services" />
+      <PackageReference Include="contentful.aspnetcore" Version="6.0.7" />
```

Comment:
Do we need this package reference?

Reply:
yeah it's used to deserialise the response